src/Campaigns/Migrations/MigrateFormsToCampaignForms.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Give\Campaigns\Migrations;
+
+use DateTime;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\ValueObjects\CampaignStatus;
+use Give\Campaigns\ValueObjects\CampaignType;
+use Give\DonationForms\Models\DonationForm;
+use Give\DonationForms\ValueObjects\DonationFormStatus;
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class MigrateFormsToCampaignForms extends Migration
+{
+    /**
+     * @inheritDoc
+     */
+    public static function id(): string
+    {
+        return 'migrate_forms_to_campaign_forms';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function timestamp(): int
+    {
+        return strtotime('2024-08-21');
```

Comment:
Let's change the timestamp here. This migration will run before the `give_campaign_forms` table is created.
See https://github.com/impress-org/givewp/blob/0b24a193d7b2db345436af43c3495c050036fd1c/src/Campaigns/Migrations/Tables/CreateCampaignFormsTable.php#L37

Reply:
Resolved, see [b2e4867](https://github.com/impress-org/givewp/pull/7532/commits/b2e4867bd42893286e3b9f4c88b41788cac25de9).